tests/test_bookmarks.py

```diff
@@ -12,6 +12,13 @@ def name(self):
         return "tap_tester_asana_bookmarks_test"
 
     def test_run(self):
+        # running sync with multiple date timestamps accross different streams due to differences in bookmark values
+        self.run_test("2021-11-09T00:00:00Z", "2023-11-10T00:00:00Z", {"projects",})
+        self.run_test("2023-11-28T00:00:00Z", "2023-11-30T00:00:00Z", {"subtasks",})
+        self.run_test("2019-01-28T00:00:00Z", "2023-11-30T00:00:00Z", self.expected_streams() - {"subtasks","projects"})
```

Comment:
Any reason, third execution doesn't have subtasks and projects streams?

Reply:
all streams do not contain data in a singular date range, hence to validate bookmarking for all streams the tests are being run with different date ranges for these two specific streams